db/seeds.rb

```diff
@@ -15,6 +15,14 @@
   vaccine.name = 'Pfizer COVID-19 Vaccine'
   vaccine.doses_required = 2
 end
+Vaccine.find_or_create_by(code: '210') do |vaccine|
+  vaccine.name = 'AstraZeneca COVID-19 Vaccine'
+  vaccine.doses_required = 2
+end
+Vaccine.find_or_create_by(code: '211') do |vaccine|
+  vaccine.name = 'Novavax COVID-19 Vaccine'
+  vaccine.doses_required = 2
+end
```

Comment:
Are we purposefully we leaving out the Non-US Covid vaccines in 
https://www.cdc.gov/vaccines/programs/iis/COVID-19-related-codes.html#fact-sheets

Reply:
I was originally going to do that, but after talking to @masnick I'll update this to include them all

Reply:
Discussed this on the track this morning, and I'm including all with a trade name. We'll make it more complete when we refactor